src/simplnx/Utilities/Math/GeometryMath.hpp

```diff
@@ -536,18 +689,23 @@ char IsPointInPolyhedron(const nx::core::TriangleGeom& faces, const std::vector<
     return 'o';
   }
 
-  std::random_device randomDevice;           // Will be used to obtain a seed for the random number engine
-  std::mt19937_64 generator(randomDevice()); // Standard mersenne_twister_engine seeded with rd()
-  std::mt19937_64::result_type seed = static_cast<std::mt19937_64::result_type>(std::chrono::steady_clock::now().time_since_epoch().count());
-  generator.seed(seed);
+  // Standard mersenne_twister_engine random seed
+  std::mt19937_64 generator(std::mt19937_64::default_seed);
```

Comment:
Does this need to actually be random? Because the default seed is fixed. So it will always return the same values.

Reply:
This gives the most correct output of the 3 options available.